content/en/security_monitoring/_index.md

```diff
@@ -1,31 +1,33 @@
 ---
-title: Security Monitoring
+title: Security and Compliance Monitoring
 kind: documentation
 ---
 
-## Overview
+<div class="alert alert-warning">
+Compliance Monitoring is currently available for private beta. Use this <a href="https://docs.google.com/forms/d/e/1FAIpQLScTA913tNGptcAeNNdWlvgECjekYoDVJwR-OkMtZYnJnq-FWw/viewform">form</a> to request access.
+</div>
 
-Datadog Security Monitoring unifies the developers, operations, and security teams into one platform. A single dashboard displays devops content, business metrics, and security content. 
+## Overview
 
 {{< img src="security_monitoring/overview_top.png" alt="Datadog Security Monitoring" >}}
 
-Datadog detects threats out-of-the-box and can notify your team of security issues by email, Slack, Jira, PagerDuty, or a webhook. 
+Datadog Security and Compliance Monitoring unify developer, operation, and security teams through one platform. Use a single dashboard to display devops content, business metrics, and security content. Detect threats, like a targeted attack, an IP communicating with your systems which matches a threat intel list, or an insecure configuration, to your application and infrastructure in real time, and notify your team of security issues by email, slack, Jira, PagerDuty, or a webhook.
 
 {{< img src="security_monitoring/takeover_ex.png" alt="Slack Example"  style="width:75%;">}}
 
-Datadog detects many different sorts of threats to your application or infrastructure. A few examples include a targeted attack, an IP communicating with your systems which matches a threat intel list, or an insecure configuration. These threats are surfaced in Datadog as Security Signals and can be correlated and triaged in the [Security Signals Explorer][1]. 
+Threats are surfaced in Datadog as Security Signals and can be correlated and triaged in the [Security Signals Explorer][1]. Security Signals are generated by Datadog Security Monitoring with [Detection Rules][2]. Detection Rules detect threats across different sources and are available out of the box for immediate use. Any of the provided detection rules can be cloned to change the configuration. A [new rule][3] can also be created from scratch.
```

Comment:
```suggestion
Threats are surfaced in Datadog as Security Signals and can be correlated and triaged in the [Security Signals Explorer][1]. Security Signals are generated by Datadog Security Monitoring with [Detection Rules][2]. Detection Rules detect threats across different sources and are available out of the box for immediate use. You can clone any of the provided detection rules to change the configuration. You can also a [new rule][3] from scratch to fit your specific use case.
```
when I read this the first time I didn't realize it was giving the user options they can take.